Tasks/AzureKeyVaultV1/task.json

```diff
@@ -51,6 +51,17 @@
             "options": {
                 "EditableOptions": "True"
             }
+        },
+        {
+            "name": "RunAsPreJob",
+            "type": "boolean",
+            "label": "Run as a pre-job",
+            "defaultValue": false,
+            "required": true,
+            "helpMarkDown": "Run the task as a pre-job. Use only if there are no untrusted tasks in the pipeline.",
```

Comment:
I'd probably change the label and helpMarkDown here a little bit since pre-job might not be a concept the task consumer is familiar with (most customers who consume tasks don't write tasks). Maybe something like:

```
"label": "Make secrets available to whole job"
"helpMarkDown": "Run the task before job execution begins. Exposes secrets to all tasks in the job, not just tasks that follow this one."

Reply:
Changed these fields in accordance with your comment.

Reply:
Thanks! This PR looks great to me!